tests/ConverterTest.php

```diff
@@ -32,12 +33,12 @@ private function defaultConfig(): array
         ];
     }
 
-    private function defaultContact(): stdClass
+    private function defaultContact(): object
```

Comment:
object type hint was only introduced with php 7.2.

Also- for most other functions- this is omitted as well.

Reply:
pls fix